langchain/chains/base.py

```diff
@@ -373,3 +555,10 @@ def save(self, file_path: Union[Path, str]) -> None:
                 yaml.dump(chain_dict, f, default_flow_style=False)
         else:
             raise ValueError(f"{save_path} must be json or yaml")
+
+    def apply(
+        self, input_list: List[Dict[str, Any]], callbacks: Callbacks = None
+    ) -> List[Dict[str, str]]:
+        """*DEPRECATED* Call the chain on all inputs in the list."""
+        logger.info("Method `Chain.apply` is deprecated.")
```

Comment:
The deprecation message is a warning, no? 
We also use `warnings.warn(` in this file. So, it could also be used here.